types/cornerstone-core/index.d.ts

```diff
@@ -0,0 +1,1517 @@
+ * @param options Options to be passed to the Image Loader
+ * @returns An Object which can be used to act after an image is loaded or loading fails
+ */
+export function loadImageFromImageLoader(imageId: string, options?: object): ImageLoadObject;
+
+/**
+ * Loads an image given an imageId and optional priority and returns a promise which will resolve to the loaded image object or fail if an error occurred. The loaded image is not stored in the cache.
+ *
+ * @param imageId A Cornerstone Image Object's imageId
+ * @param options Options to be passed to the Image Loader
+ */
+export function loadImage(imageId: string, options?: object): Promise<Image>;
+
+/**
+ * Loads an image given an imageId and optional priority and returns a promise which will resolve to the loaded image object or fail if an error occurred. The image is stored in the cache.
+ *
+ * @param imageId A Cornerstone Image Object's imageId
+ * @param options Options to be passed to the Image Loader
+ * @returns Image Loader Object
+ */
+export function loadAndCacheImage(imageId: string, options?: object): ImageLoadObject;
+
+/**
+ * Registers an imageLoader plugin with cornerstone for the specified scheme
+ *
+ * @param scheme The scheme to use for this image loader (e.g. 'dicomweb', 'wadouri', 'http')
+ * @param imageLoader A Cornerstone Image Loader function
+ */
+export function registerImageLoader(scheme: string, imageLoader: __UnknownFunction__): void;
+
+/**
+ * Registers a new unknownImageLoader and returns the previous one
+ *
+ * @param imageLoader A Cornerstone Image Loader
+ * @returns The previous Unknown Image Loader
+ */
+export function registerUnknownImageLoader(imageLoader: __UnknownFunction__): __UnknownFunction__ | undefined;
+
+// ================================================================
+// ImageCache
+// ================================================================
+
+// ================================================================
+// Metadata
+// ================================================================
+/**
+ * Adds a metadata provider with the specified priority
+ *
+ * @param provider Metadata provider function
+ * @param priority 0 is default/normal, > 0 is high, < 0 is low (optional, default 0)
+ */
+export function addProvider(provider: __UnknownFunction__, priority: number): void;
+
+/**
+ * Removes the specified provider
+ *
+ * @param provider Metadata provider function
+ */
+export function removeProvider(provider: __UnknownFunction__): void;
+
+/**
+ * Gets metadata from the registered metadata providers. Will call each one from highest priority to lowest until one responds
+ *
+ * @param type The type of metadata requested from the metadata store
+ * @param imageId The Cornerstone Image Object's imageId
+ * @returns The metadata retrieved from the metadata store
+ *
+ * TODO check clearitify the unknown type
+ */
+export function getMetaData(type: string, imageId: string): unknown;
```

Comment:
To confirm, the expectation is that the result of `getMetaData` is intended to be difficult to consume without an explicit cast?